pages/auth/logout.tsx

```diff
@@ -1,22 +1,27 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "next-i18next";
 import Link from "next/link";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import { GetServerSideProps } from "next";
-import { useSession } from "next-auth/react";
 
 const Logout = () => {
   const { i18n, t } = useTranslation("logout");
-  const { data: session } = useSession();
+  const [lastLoginTime, setLastLoginTime] = useState("");
+
+  useEffect(() => {
+    setLastLoginTime(
+      new Date().toLocaleString(`${i18n.language + "-CA"}`, {
```

Comment:
Technically, it won't be the same date value as the last LOGOUT value in the `AccessLogs` table, but I suppose it'll be close enough that it won't matter.